js/everything.js

```diff
@@ -281,6 +337,35 @@ var ZenPlayer = {
             toggleElement(event, ".plyr__video-wrapper", "Player");
         });
     },
+    setupAutoplayToggle: function()
+    {
+        // toggle auto next song playing
+        $("#toggleAutoplay").click(function(event)
+        {
+            var toggleTextElement = $("#" + event.currentTarget.id);
+            if (autoplayState === null)
```

Comment:
This code can be simplified by using an if condition of `!!autoplayState` which converts the value to a boolean, `null` will evaluate to `false`.

ie:

```shell
> var vals = [null, false, true];
undefined
> for (var v in vals) { console.log(!!vals[v]); }
false
false
true
```